Repository/ServiceEntityRepository.php

```diff
@@ -8,46 +8,8 @@
 
 if (property_exists(EntityRepository::class, '_entityName')) {
     // ORM 2
-    /**
-     * Optional EntityRepository base class with a simplified constructor (for autowiring).
-     *
-     * To use in your class, inject the "registry" service and call
-     * the parent constructor. For example:
-     *
-     * class YourEntityRepository extends ServiceEntityRepository
-     * {
-     *     public function __construct(ManagerRegistry $registry)
-     *     {
-     *         parent::__construct($registry, YourEntity::class);
-     *     }
-     * }
-     *
-     * @template T of object
-     * @template-extends LazyServiceEntityRepository<T>
-     */
-    class ServiceEntityRepository extends LazyServiceEntityRepository
-    {
-    }
+    class_alias(ServiceEntityRepositoryOrm2::class, 'ServiceEntityRepository');
```

Comment:
This call is broken. The second argument should be the class name, which must not be `ServiceEntityRepository` but `Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository` (which can be written as `ServiceEntityRepository::class`)

Reply:
Hi @stof, I will prepare everything in 2-3 days. I just created a PR for you to look at and give comments before I started working on it. The problem here is that some analyzers cannot process this class correctly and this issue needs to be corrected somehow. Psalm, PHPStan and in general the tools of PHPStorm itself do not process correctly (phpcs too). I need to understand whether the problem is really in the code or in the handlers.